commons/src/main/java/com/powsybl/openrao/commons/NumberRounding.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.openrao.commons;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+/**
+ * @author Thomas Bouquet {@literal <thomas.bouquet at rte-france.com>}
+ */
+public final class NumberRounding {
+    private static final double ROUNDING_EPSILON = 1e-8; // "noise" required to avoid side effects when rounding negative powers of 10
+
+    private NumberRounding() {
+    }
+
+    public static int computeNumberOfRelevantDecimals(double value, int defaultDecimals) {
```

Comment:
as tu regardé si la fonction scale pouvait convenir ?

Reply:
I did and there seem to be undesired behaviors on negative powers of ten

Reply:
more generaly, scale computes the number of decimals of a double whereas we are interested by the position of the first non-null decimal